src/main.cpp

```diff
@@ -306,27 +298,21 @@ int main(int argc, char* argv[]) {
           throw std::invalid_argument("Unknown interface\n\n");
       }
     }
-    else if(!arguments.map_file_path.empty()) // MAP mode
+    else if(arguments.device_name_raw) // CDD mode
```

Comment:
Above it is called "MAP" mode, which I think is correct. Also in direct mode the [device] argument should be a CDD, while in DMAP mode it is an alias. Is simply /dev/pcieunis2 still allowed when opening a device? At least it's deprecated.

Reply:
True, I will change the comment.